lib/ansible/modules/cloud/vmware/vmware_object_role_permission.py

```diff
@@ -220,6 +221,11 @@ def get_object(self):
                 msg="Specified object %s of type %s was not found."
                 % (self.params['object_name'], self.params['object_type'])
             )
+        if self.params['object_type'] == 'DistributedVirtualSwitch':
+            msg = "You have specified Distributed vSwitch but permissions for a distributed switch " \
```

Comment:
Does this mean the permissions for the switch should be created using a different module? If so, could you include the module names in the error message? Something like "You are specifying a Distributed vSwitch in the vmware_object_role_permission module. Permissions for a distributed switch must be managed with the ???? or ???? module."

Reply:
No, vmware_object_role_permission module itself manages the permission for Distributed vSwitch and Network folder (parent of Distributed vSwitch).

VMware has this limitation/feature where if you are specifying permission on distributed vswitch, you need to specify permission on Network folder (i.e. parent folder of distributed vswitch) and then it will propagate the permission to its children. 

Please suggest me better wording for this line. 

Thanks for reviewing.